
real	1m6.067s
user	0m55.635s
sys	0m4.185s

real	2m52.245s
user	2m43.784s
sys	0m2.659s

real	1m23.155s
user	1m16.738s
sys	0m1.989s

real	2m41.375s
user	2m30.733s
sys	0m3.426s

real	1m36.650s
user	1m21.065s
sys	0m8.391s

real	3m5.155s
user	2m50.099s
sys	0m4.851s

real	1m13.312s
user	1m6.201s
sys	0m2.435s

real	2m33.727s
user	2m24.230s
sys	0m3.048s

real	1m38.045s
user	1m26.371s
sys	0m5.432s

real	2m39.159s
user	2m29.821s
sys	0m3.142s
Traceback (most recent call last):
  File "/home/thiago.esterci/2D_imune_edema_pinn/edo_pinn_model.py", line 187, in <module>
    Cl = pk.load(f)
EOFError: Ran out of input

real	0m2.274s
user	0m1.451s
sys	0m0.198s

real	2m41.720s
user	2m31.385s
sys	0m3.221s

real	1m3.405s
user	0m58.266s
sys	0m1.901s

real	2m57.482s
user	2m47.570s
sys	0m3.226s

real	2m0.950s
user	1m40.133s
sys	0m11.636s

real	2m45.098s
user	2m35.671s
sys	0m3.108s

real	1m41.752s
user	1m19.571s
sys	0m13.635s

real	3m45.424s
user	3m11.059s
sys	0m23.037s

real	0m46.429s
user	0m42.028s
sys	0m1.829s

real	2m30.920s
user	2m23.023s
sys	0m2.545s
